proto/stakeibc/tx.proto

```diff
@@ -65,6 +65,7 @@ message MsgRegisterHostZone{
   string base_denom = 4 [ (gogoproto.moretags) = "yaml:\"base_denom\"" ];
   string local_denom = 5 [ (gogoproto.moretags) = "yaml:\"local_denom\"" ];
   string creator = 6;
+  string channel_id = 10 [ (gogoproto.moretags) = "yaml:\"channel_id\"" ];
```

Comment:
why should we pass the channel into registerHostZone? Does this replace having the channel assigned automatically?

Reply:
transfer channel so we know what channel to move the tokens over on. I don't think we strictly need this (we could back out the channel based on the denom trace when we create the DepositRecord), but I think it's slightly more standard to only support one transfer channel (I believe it's similar to Osmosis)